src/api/app/controllers/application_controller.rb

```diff
@@ -569,6 +569,17 @@ def set_response_format_to_xml
     request.format = :xml if request.format == :html
   end
 
+  def extract_user_public
```

Comment:
Actually you're correct. This method is only ever used in a `before_action` so the returns are pointless..